### DISCUSSION TITLE
problems on token

### DISCUSSION THREAD

**Discussion by Niuhao1998** [hvac/hvac#1064] [Q&A]
I tried to use hvac to create a client without token, then I find hvac does not help me to initialize a token.
so,I tried to give it a token "123456",however there is some bugs here.
![image](https://user-images.githubusercontent.com/26596966/203730130-f84c0f49-bc9a-4887-8804-4d80ac5ade43.png)

![image](https://user-images.githubusercontent.com/26596966/203730434-8863ad55-5acb-4045-ac98-7c73ac64d58b.png)

 raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='localhost', port=8200): Max retries exceeded with url: /v1/auth/token/lookup-self (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000186366784F0>: Failed to establish a new connection: [WinError 10061] Unable to connect due to aggressive rejection by the target computer.'))

**Comment by briantist** [comment]
Hi @Niuhao1998 I would need more information about what you're trying to do and ow it is not working. It would be more helpful to have code and output as text rather than images.

The error `Unable to connect due to aggressive rejection by the target computer` is something I have not seen before, but this is a response that's coming from your Vault server or your local network stack; I do not think it will be solvable in the `hvac` library.
